### ISSUE TITLE
Documentation: How do I load nuget references from alternative package sources?

### ISSUE THREAD

**Issue by KevM** [dotnet/interactive#698]
**Is your feature request related to a problem? Please describe.**
Trying to use dotnet interactive to create c# notebooks. I need to reference a package on a private Nuget feed. Looking at the documentation I do not see how to do this.

* I tried including a `nuget.config` in the notebook folder. 
* I tried various variations on adding source to the `#r` command.

```
#r "nuget:My.Private.Nuget -Source https://www.myget.org/F/{feed}/api/v3/index.json"
```

Result:
> No packages exist with this id in source(s): Microsoft Visual Studio Offline Packages, nuget.org

**Describe the solution you'd like**
Update [dotnet interactive nuget docs](https://github.com/dotnet/interactive/blob/master/docs/nuget-overview.md) to communicate how to do more than just reference a nuget by id and version.

**Describe alternatives you've considered**
I also looked for details on how the `#r` command is implemented but I do not see enough in the architecture docs to get there. Sorry if I missed something.

**Comment by zyzhu** [comment]
Reference similar old issue in fsharp repo. https://github.com/dotnet/fsharp/issues/7834

**Comment by dsyme** [comment]
I believe it is 

    #i "nuget: https://ci.appveyor.com/nuget/diffsharp"
    #r "nuget: DiffSharp-lite,0.9.5-local-200721"

**Comment by jonsequitur** [comment]
> I tried including a nuget.config in the notebook folder.

This is seemingly intuitive and has been suggested before. I've opened #702 for it.

**Comment by zyzhu** [comment]
Just to document what I found today.

I've tried to follow steps in the following link to add private feed with password using nuget command. My private repo is added to global nuget repository in local computer.
https://docs.microsoft.com/en-us/azure/devops/artifacts/nuget/nuget-exe?view=azure-devops
```nuget sources add -name {your feed name} -source {your feed URL} -username {anything} -password {your PAT}```
Then ```#r "nuget:package"``` works now in notebook.

There is no need to add ```"#i nuget url"``` if configured like this as notebook checks global nuget settings now.
